roles/docker/tasks/systemcontainer_docker.yml

```diff
@@ -165,6 +165,6 @@
   delay: 30
 
 - set_fact:
-    docker_service_status_changed: r_docker_systemcontainer_docker_start_result | changed
+    docker_service_status_changed: "{{ r_docker_systemcontainer_docker_start_result | changed }}"
```

Comment:
@mtnbikenc I believe without this change it was simply storing the string "r_docker_systemcontainer_docker_start_result | changed" in the fact based on logs, is that correct?